crypto/src/main/java/org/hyperledger/besu/crypto/PrivateKey.java

```diff
@@ -0,0 +1,74 @@
+package org.hyperledger.besu.crypto;
+
+import org.apache.tuweni.bytes.Bytes32;
+import org.apache.tuweni.units.bigints.UInt256;
+
+import java.math.BigInteger;
+import java.util.Objects;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+public class PrivateKey implements java.security.PrivateKey {
```

Comment:
Name should be more descriptive, like `SECPPrivateKey.`

Reply:
Good suggestion. I've renamed it.